src/test/java/ezschedule/logic/commands/RecurCommandTest.java

```diff
@@ -0,0 +1,313 @@
+
+        String expectedMonthFeb = "February";
+        String actualMonthFeb = validDailyRecurCommandStub.intToStringMonth(2);
+
+        String expectedMonthMar = "March";
+        String actualMonthMar = validDailyRecurCommandStub.intToStringMonth(3);
+
+        String expectedMonthApr = "April";
+        String actualMonthApr = validDailyRecurCommandStub.intToStringMonth(4);
+
+        String expectedMonthMay = "May";
+        String actualMonthMay = validDailyRecurCommandStub.intToStringMonth(5);
+
+        String expectedMonthJun = "June";
+        String actualMonthJun = validDailyRecurCommandStub.intToStringMonth(6);
+
+        String expectedMonthJul = "July";
+        String actualMonthJul = validDailyRecurCommandStub.intToStringMonth(7);
+
+        String expectedMonthAug = "August";
+        String actualMonthAug = validDailyRecurCommandStub.intToStringMonth(8);
+
+        String expectedMonthSep = "September";
+        String actualMonthSep = validDailyRecurCommandStub.intToStringMonth(9);
+
+        String expectedMonthOct = "October";
+        String actualMonthOct = validDailyRecurCommandStub.intToStringMonth(10);
+
+        String expectedMonthNov = "November";
+        String actualMonthNov = validDailyRecurCommandStub.intToStringMonth(11);
+
+        String expectedMonthDec = "December";
+        String actualMonthDec = validDailyRecurCommandStub.intToStringMonth(12);
+
+        assertTrue(expectedMonthJan.equals(actualMonthJan));
+        assertTrue(expectedMonthFeb.equals(actualMonthFeb));
+        assertTrue(expectedMonthMar.equals(actualMonthMar));
+        assertTrue(expectedMonthApr.equals(actualMonthApr));
+        assertTrue(expectedMonthMay.equals(actualMonthMay));
+        assertTrue(expectedMonthJun.equals(actualMonthJun));
+        assertTrue(expectedMonthJul.equals(actualMonthJul));
+        assertTrue(expectedMonthAug.equals(actualMonthAug));
+        assertTrue(expectedMonthSep.equals(actualMonthSep));
+        assertTrue(expectedMonthOct.equals(actualMonthOct));
+        assertTrue(expectedMonthNov.equals(actualMonthNov));
+        assertTrue(expectedMonthDec.equals(actualMonthDec));
+    }
+
+    @Test
+    public void execute_invalidIntToStringMonths_success() {
+        String invalidMessage = "No such month";
+
+        // invalid months [..., 0]
+        String actualMonthZero = validDailyRecurCommandStub.intToStringMonth(0);
+        String actualMonthNegative = validDailyRecurCommandStub.intToStringMonth(-10);
+
+        // invalid months [13, ...]
+        String actualMonthThirteen = validDailyRecurCommandStub.intToStringMonth(13);
+        String actualMonthHundred = validDailyRecurCommandStub.intToStringMonth(100);
+
+
+        assertTrue(invalidMessage.equals(actualMonthZero));
+        assertTrue(invalidMessage.equals(actualMonthNegative));
+        assertTrue(invalidMessage.equals(actualMonthThirteen));
+        assertTrue(invalidMessage.equals(actualMonthHundred));
```

Comment:
Can use `assertEquals` here?

Reply:
@JrmCkh advice on usage of assertTrue vs assertEquals ?

Reply:
Can use assertEquals. We are not testing on equals() method, which we have tested in a separate test case.

Reply:
updated